hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/impl/AbstractWipePrecompile.java

```diff
@@ -0,0 +1,117 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.services.store.contracts.precompile.impl;
+
+import static com.hedera.services.exceptions.ValidationUtils.validateTrue;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_SIGNATURE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.TokenType.NON_FUNGIBLE_UNIQUE;
+
+import com.hedera.services.context.SideEffectsTracker;
+import com.hedera.services.contracts.sources.EvmSigsVerifier;
+import com.hedera.services.ledger.accounts.ContractAliases;
+import com.hedera.services.store.contracts.WorldLedgers;
+import com.hedera.services.store.contracts.precompile.InfrastructureFactory;
+import com.hedera.services.store.contracts.precompile.SyntheticTxnFactory;
+import com.hedera.services.store.contracts.precompile.codec.DecodingFacade;
+import com.hedera.services.store.contracts.precompile.codec.WipeWrapper;
+import com.hedera.services.store.contracts.precompile.utils.KeyActivationUtils;
+import com.hedera.services.store.contracts.precompile.utils.PrecompilePricingUtils;
+import com.hedera.services.store.models.Id;
+import com.hederahashgraph.api.proto.java.Timestamp;
+import com.hederahashgraph.api.proto.java.TransactionBody;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.UnaryOperator;
+import org.apache.tuweni.bytes.Bytes;
+import org.hyperledger.besu.evm.frame.MessageFrame;
+
+public class AbstractWipePrecompile extends AbstractWritePrecompile {
+
+    protected final ContractAliases aliases;
+    protected final EvmSigsVerifier sigsVerifier;
+    protected WipeWrapper wipeOp;
+    private static final List<Long> NO_SERIAL_NOS = Collections.emptyList();
+
+    public AbstractWipePrecompile(
+            WorldLedgers ledgers,
+            DecodingFacade decoder,
+            final ContractAliases aliases,
+            final EvmSigsVerifier sigsVerifier,
+            SideEffectsTracker sideEffects,
+            SyntheticTxnFactory syntheticTxnFactory,
+            InfrastructureFactory infrastructureFactory,
+            PrecompilePricingUtils pricingUtils) {
+        super(
+                ledgers,
+                decoder,
+                sideEffects,
+                syntheticTxnFactory,
+                infrastructureFactory,
+                pricingUtils);
+        this.aliases = aliases;
+        this.sigsVerifier = sigsVerifier;
+    }
+
+    @Override
+    public TransactionBody.Builder body(Bytes input, UnaryOperator<byte[]> aliasResolver) {
+        return null;
+    }
+
+    @Override
+    public long getMinimumFeeInTinybars(Timestamp consensusTime) {
+        return 0;
```

Comment:
Q: Why override just to set 0? Why not just leave method out since extending class will implement?